proposals/LP0004-project-governance.md

```diff
@@ -0,0 +1,393 @@
+
+Voting begins the fourth Monday in January and continues for 2 weeks. Election results will be announced no later than the second Monday in February. The term of the newly elected _area team_ begins the first Monday in March. Each _area team_ will meet during the first week in March to elect from themselves the team chair to re-constitute the _project council_.
+
+### Mandatory Voting
+
+To encourage participation in elections voting is considered mandatory. Any _eligible voting contributor_ who does not vote in an election will be ineligible to vote in the next election and their registration will be removed if they do not vote in the next election they are eligible to vote in.
+
+## Amending This Document
+
+This document should be amended through public proposals following the [LP0001 LLVM Decision Making](https://github.com/llvm/llvm-www/blob/main/proposals/LP0001-LLVMDecisionMaking.md) process. The _project council_ shall serve as the review managers for any proposal to amend this document.
+
+## Impact On Other Projects
+
+This proposal impacts all LLVM projects and all contributors to the LLVM community.
+
+## Frequently Asked Questions
+
+If there were common questions that came up in the pitch phase, please summarize them and what you think the answer is.  Doing so gives you a chance to address them before the formal review starts up, which can lead to a more productive review thread - one that spends less time rehashing points that have already come up.
+
+## Appendix 1: LLVM Community Metrics Methods
+
+This document references some LLVM community metrics. This appendix contains the methodology and tools used to produce the metrics cited above. The metrics are gathered from git metadata with a scoped date range of September 1, 2022 through September 1, 2023. Using a specific date range allows these metrics to be re-generated consistently, however the metrics should be extended to encompass the latest usage patterns when driving decision making.
+
+### git-crawler.py
+
+This script traverses the directories in the LLVM project listing all the authors of commits in the directory once per commit. The script sums up the number of times an authors name appears to produce an approximation of the number of unique authors who have contributed code under a specific sub-directory. The script does not traverse into sub-directories that have less than 50 unique authors, and it skips `test` and `unittests` folders.
+
+The output printed is the number of unique authors that contribute to each directory with 50 or more authors, and a list of the top ten authors that committed to that directory with their number of commits.
+
+```python
+#!/usr/bin/env python3
+
+import os
+import subprocess
+from collections import Counter
+
+def handleDirectory(path):
+ path = path.replace(os.getcwd() + '/', ':')
```

Comment:
Running this locally on `lldb/`, there is a `__pycache__` folder that is a result of running the test suite. So I get `fatal: Unimplemented pathspec magic '_' in ':__pycache__'`.

This is not a folder git tracks, so maybe you could instruct git to ignore those somehow?